src/main/java/slbedu/library/services/UserManager.java

```diff
@@ -56,4 +56,13 @@ public Response isAuthenticated() {
         return RESPONSE_OK;
     }
 
+    @Path("getuser")
```

Comment:
Following the current conventions, the name of this path should be 'current' rather than 'getuser'. So that finally the whole URI is rest/user/current